docs/ui.md

```diff
@@ -17,4 +17,28 @@ You can also use other web servers to serve the contents of the www/app director
 ### Running remotely
 When Kubernetes is deployed remotely, the UI is deployed as a cluster addon. To access it, visit `/ui`, which redirects to `/api/v1/proxy/namespaces/default/services/kube-ui/#/dashboard/`, on your master server.
 
+## Using the UI
```

Comment:
[Comment on "See the [README](..//www/README.md) in the www directory for more information" from line 4 -- unfortunately it's not possible to directly comment on lines that aren't changed in the PR]

I would suggest moving this sentence to a new section at the end of the doc, something ilke

```
## More information

For more information, see the [README](../www/README.md) in the www directory for more information. 
```

I don't think it's a good idea for people to jump over to that page until they've gone through everything on this page. That page is super-confusing and I'm not sure all of it is even completely accurate anymore.

[Comment on Running locally / Running remotely]

I would suggest flipping the order of these two sections -- I think the more common case is that people will be running the cluster remotely, so better to put that one first.